src/VisualStudio/Core/Impl/Options/GridOptionPreviewControl.xaml.cs

```
@@ -93,5 +106,264 @@ internal override void Close()
                 this.ViewModel.Dispose();
             }
         }
+
+        internal void Generate_Save_Editorconfig(object sender, System.Windows.RoutedEventArgs e)
+        {
+            var optionSet = this.ViewModel.ApplyChangedOptions(this.OptionService.GetOptions());
+            var editorconfig = new StringBuilder();
+            Generate_Editorconfig(optionSet, _language, editorconfig, _getLangaugeSpecificEditorConfigOptions);
+            var sfd = new System.Windows.Forms.SaveFileDialog
+            {
+                Filter = "All files (*.*)|",
+                FileName = ".editorconfig",
+                Title = "Save .editorconfig file"
+            };
+            if (sfd.ShowDialog() == System.Windows.Forms.DialogResult.OK)
+            {
+                IOUtilities.PerformIO(() =>
```

should we silence silence all errors instead of exposing something in the UI?